Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -63,11 +63,42 @@
 FETCH_QUERY = RAW_QUERY or FETCH_QUERY_PARM
 
 
-def convert_date_to_timestamp(date):
+def prepare_datetime_format(datetime_format: str):
+    """Converting the format from ES to Python so that the Arrow format function will format it correctly.
+
+    Args:
+        datetime_format: An ES date time format for example 'yyyy-MM-dd HH:mm:ss'.
+        see here for more examples: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html#built-in-date-formats.
+
+    Returns:
+        A datetime format in python from 'yyyy-MM-dd HH:mm:ss' => 'YYYY-MM-DD HH:mm:ss'.
+    """
+    return datetime_format.replace('yy', 'YY').replace('dd', 'DD')
+
+
+def get_datetime_field_format(es: Elasticsearch, index: str = FETCH_INDEX, field: str = TIME_FIELD):
+    """Returns the datetime format of a field in an index.
+
+    Args:
+        es: An ES object.
+        index: The index from which to return the mapper.
+        field: The field from which to return the format.
+
+    Returns:
+        String representing the date time format for example 'yyyy-MM-dd HH:mm:ss'.
+    """
+    mapping = es.indices.get_mapping(index=index)
+    datetime_field = demisto.get(mapping, f'{index}.mappings.properties.{field}', {})
+
+    return datetime_field.get('format', 'YYYY-MM-DD HH:mm:ss')
```

Comment:
Put `YYYY-MM-DD HH:mm:ss` in a constant, it's used in several places in the code (e.g. in the definition of `convert_date_to_timestamp` and in `get_time_range`)